src/main/java/LacksDataBaseProject/LocalRepository.java

```diff
@@ -0,0 +1,29 @@
+package LacksDataBaseProject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class LocalRepository<T extends Object> implements CRUDRepository<T> {
+
+    private List<T> list = new ArrayList<>();
+
+    @Override
+    public void add(T t) {
+        System.out.println(String.format("Add %s in: %s", t, getRepositoryName()));
```

Comment:
Tutaj też proponuje się pozbyć zmiennych stałych :) jeśli chcecie coś logować to proponuje wykorzystać jakiś logger, np: https://github.com/JakeWharton/timber . Dajcie znać czy mieliście coś o loggerach, bo jeśli nie podeślę więcej info :)

Reply:
OK. Spróbuje używać tego: https://www.slf4j.org/, bo ludzie to mocno propsują, ale potrzebuję trochę czasu żeby to ogarnąć :)

Reply:
spoko :) Timber jest na Androidzie wykorzystywany dość często :) w razie czego daj znać :)